Makefile

```diff
@@ -83,6 +96,28 @@ all ::
 	@echo "HTML built: `ls -lAFgh $(BUILD_DIR)/manual/build/html/index.html`"
 	@echo "PDF built: `ls -lAFgh $(BUILD_DIR)/manual/build/latex/nexus.pdf`"
 
+$(BASE_CLASS_DIR)/%.nxdl.xml : $(BASE_CLASS_DIR)/$(NYAML_SUBDIR)/%.yaml
+	$(PYTHON) -m dev_tools.nyaml2nxdl.nyaml2nxdl --input-file $<
+	mv $(BASE_CLASS_DIR)/$(NYAML_SUBDIR)/$*.nxdl.xml $@
+
+$(CONTRIB_DIR)/%.nxdl.xml : $(CONTRIB_DIR)/$(NYAML_SUBDIR)/%.yaml
+	$(PYTHON) -m dev_tools.nyaml2nxdl.nyaml2nxdl --input-file $<
+	mv $(CONTRIB_DIR)/$(NYAML_SUBDIR)/$*.nxdl.xml $@
+
+$(APPDEF_DIR)/%.nxdl.xml : $(APPDEF_DIR)/$(NYAML_SUBDIR)/%.yaml
+	$(PYTHON) -m dev_tools.nyaml2nxdl.nyaml2nxdl --input-file $<
+	mv $(APPDEF_DIR)/$(NYAML_SUBDIR)/$*.nxdl.xml $@
+
+NXDLS := $(foreach dir,$(NXDL_DIRS),$(wildcard $(dir)/*.nxdl.xml))
+nyaml : $(DIRS)
```

Comment:
Where is `DIRS` defined?

Reply:
It isn't. It's a typo and should read NXDL_DIRS. However, I think it's the better choice to leave out the dependencies entirely here (we can not add the nxdls here because then we end up with a cyclic dependency - hence the for loop). So I will just remove this and the unused `NXDL_DIRS` on top of the file.

Reply:
It's fixed. However, I'm not allowed to resolve the issue myself.